mula/scheduler/schedulers/schedulers/boefje.py

```diff
@@ -343,247 +233,168 @@ def push_tasks_for_rescheduling(self):
                     ]
                 )
             )
-        except storage.errors.StorageError as exc_db:
-            self.logger.error(
-                "Could not get schedules for rescheduling %s",
-                self.scheduler_id,
-                scheduler_id=self.scheduler_id,
-                organisation_id=self.organisation.id,
-                exc_info=exc_db,
-            )
-            raise exc_db
-
-        if not schedules:
-            self.logger.debug(
-                "No schedules tasks found for scheduler: %s",
-                self.scheduler_id,
-                scheduler_id=self.scheduler_id,
-                organisation_id=self.organisation.id,
-            )
+            if not schedules:
+                self.logger.debug(
+                    "No schedules tasks found for scheduler: %s", self.scheduler_id, scheduler_id=self.scheduler_id
+                )
+                return
+        except StorageError:
+            self.logger.exception("Error occurred while processing rescheduling", scheduler_id=self.scheduler_id)
             return
 
-        with futures.ThreadPoolExecutor(
-            thread_name_prefix=f"BoefjeScheduler-TPE-{self.scheduler_id}-rescheduling"
-        ) as executor:
+        with futures.ThreadPoolExecutor(thread_name_prefix=f"TPE-{self.scheduler_id}-rescheduling") as executor:
             for schedule in schedules:
-                boefje_task = BoefjeTask.model_validate(schedule.data)
-
-                # Plugin still exists?
                 try:
+                    boefje_task = models.BoefjeTask.model_validate(schedule.data)
```

Comment:
I found an issue here that the 'data' field in the schedule contains the uuid of the boefje_meta, which is passed all the way to bytes that has a uniqueness constraint on this id. Hence, when we create new boefje tasks from the `data` field, we should make sure the id is a _new_ uuid, because rescheduled tasks now all fail for me.

Reply:
This is strange, it should create a `new_boefje_task` that should have a new unique uuid. Perhaps you can show me what happened?

Reply:
Update: probably an issue on my system after a rebuild, although a strange one that might have to do with the creation of raw files failing after creation of a boefje_meta succeeded from the job_handler. Resolving for now.